cmd/raidz_test/raidz_test.h

```diff
@@ -34,6 +34,8 @@ static const char *raidz_impl_names[] = {
 	"sse2",
 	"ssse3",
 	"avx2",
+	"neon",
+	"neon2",
```

Comment:
I've run raidz_test -S on a variety of hardware with no issue, since it's a pure userland binary it's much easier.
-v is not much use, since the raidz_test is multithreaded... and some of those systems have a lot of cores ;-) - the output is quite mangled.